src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
I've already mentioned 2 times to make `Random` instance a constant - stopping review here.